### PR TITLE
Create a profile page

### PR DESCRIPTION
**Pull request by Omulosi**
# Description
Create a profile page for displaying user info

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

## UI

Selection_077

### CODE REVIEW

**Comment by emkayDauda** [comment]
Awesome job, John! Do you want to maybe leave the route in so we can see what it looks like on the preview link?

**Comment by Omulosi** [comment]
> Awesome job, John! Do you want to maybe leave the route in so we can see what it looks like on the preview link?

As in? not sure what you meant there...also, the page depends on some updates I made to the view profile endpoint on the backend ( basically added a createdon and image-url fields), so you'd want want to pull in those changes to test

**Review by annawinther — DISMISSED** [review]
Looks really nice, John! Awesome job! 🔥 

As we talked about in standup too; for the `change password` button, is the functionality to change the password there and working, or will we be using the `reset_password` endpoint? Maybe that is something to think about too? 😄

**Comment by Omulosi** [comment]
> Looks really nice, John! Awesome job! 
> 
> As we talked about in standup too; for the `change password` button, is the functionality to change the password there and working, or will we be using the `reset_password` endpoint? Maybe that is something to think about too?

Thanks Anna! It will be using the reset_password endpoint.

**Review by annawinther — DISMISSED** [review]
Nice job, John! 

Thanks for adding the Profile route to the nav bar as well 👍

**Review by annawinther — DISMISSED** [review]

**Review by emkayDauda — DISMISSED** [review]
Nicely done, John.

If you could just fix those merge conflicts so this can be merged easier

**Comment by ethaeral** [comment]
Screenshot from 2020-01-15 04-05-35

Your design is different from the Figma.

**Review by annawinther — COMMENTED** [review]
Looks really nice, John! A lot more like the Figma now 💪 

There is just one merge conflict to be solved 😄

**Comment by emkayDauda** [comment]
Looking really clean, John! 💪🏾💪🏾

Great work. Just the one merge conflict and this will be really sweet.

File: src/components/ProfileMainPage/ProfileMainSection.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  let period = moment(registered).fromNow();
+  period = period === 'Invalid date' ? null : period;
+
+  return (
+    <MainSection>
+      <Content>
+        <h1>About</h1>
+        <span>
+          {fullName} has been a user from {period || `unspecified`}
+        </span>
```
Comment (emkayDauda) [inline src/components/ProfileMainPage/ProfileMainSection.js]: I would also say `{fullname} has been a user for {period}` reads a little better maybe? 🤔

**Pull request merged**